sycl/test-e2e/Graph/RecordReplay/dotp_in_order_pause.cpp

```diff
@@ -0,0 +1,98 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s %}
+//
+// CHECK-NOT: LEAK
```

Comment:
There's a different leak checking invocation that's preferred https://github.com/intel/llvm/pull/12185#discussion_r1430769589
```suggestion
// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK  %}
```